srcpkgs/clion/template

```diff
@@ -0,0 +1,58 @@
+# Template file for 'clion'
+_dlname="CLion"
+_pkgname="clion"
+pkgname=clion
+version=2019.1.2
+revision=1
+archs="x86_64"
+depends="virtual?java-environment giflib libXtst"
+short_desc="CLion: A smart cross-platform IDE for C and C++"
+maintainer="Anton Afanasyev <anton@doubleasoftware.com>"
+license="Commercial"
```

Comment:
License is not [spdx](https://spdx.org/licenses/), so need to be written as `custom:Commercial` and included in package.

Reply:
Thanks!
I've put `custom:Commercial`, but I'm not sure whether I should be including it in the package.
There are about 30 license files for each of these applications, and I'm not sure it makes sense to include all?